public/app/features/alerting/unified/components/rule-editor/alert-rule-form/AlertRuleForm.tsx

```diff
@@ -101,23 +110,27 @@ export const AlertRuleForm = ({ existing, prefill }: Props) => {
     shouldFocusError: true,
   });
 
-  const { handleSubmit, watch } = formAPI;
+  const {
+    handleSubmit,
+    watch,
+    formState: { isSubmitting },
+  } = formAPI;
 
   const type = watch('type');
+  const grafanaTypeRule = type?.includes('grafana') ?? true;
```

Comment:
This should also work with the PR for adding support for Grafana-managed recording rules.